vm/src/stdlib/winapi.rs

```diff
@@ -89,12 +135,15 @@ mod _winapi {
 
     #[pyfunction]
     fn CloseHandle(handle: usize, vm: &VirtualMachine) -> PyResult<()> {
-        cvt(vm, unsafe { handleapi::CloseHandle(handle as HANDLE) }).map(drop)
+        cvt(vm, unsafe {
+            Foundation::CloseHandle(Foundation::HANDLE(handle as isize))
+        })
+        .map(drop)
     }
 
     #[pyfunction]
     fn GetStdHandle(std_handle: u32, vm: &VirtualMachine) -> PyResult<usize> {
-        cvt(vm, unsafe { processenv::GetStdHandle(std_handle) }).map(husize)
+        cvt(vm, Console::GetStdHandle(STD_HANDLE(std_handle))).map(husize)
```

Comment:
```suggestion
        let handle = unsafe { Console::GetStdHandle(STD_HANDLE(std_handle)) }.map_err(|err| vm.new_type_error(format!("{}", err.message()) ))?;
        cvt(vm, handle).map(husize)
```
This is one kind of example. please convert error type properly.